.sauce/config.yml

```diff
@@ -11,19 +13,25 @@ sauce:
       - other tag
     build: Release $CI_COMMIT_SHORT_SHA
 docker:
-  # Affects how test files are transferred to the docker container when using the docker mode (--test-env docker).
+  # Affects how test files are transferred to the docker container when using the docker mode.
   fileTransfer: copy # Choose between mount|copy.
 playwright:
   version: 1.7.1 # See https://docs.saucelabs.com/testrunner-toolkit for a list of supported versions.
 # Controls what files are available in the context of a test run (unless explicitly excluded by .sauceignore).
 rootDir: ./
 suites:
-  - name: "Firefox"
-    platformName: "Windows 10" # Only relevant when running a test against the sauce cloud (--test-env sauce).
+  - name: "Firefox using global mode" # If no `mode` setting for a suite, it should use the global setting.
```

Comment:
```suggestion
  - name: "Firefox using global mode" # Since the suite doesn't specify the `mode`, it'll inherit the mode specified via `defaults.mode` (see line number 3 and 4 of this config file)
```